exercises/intro/intro1.d

```diff
@@ -1,3 +1,14 @@
+/// Sometimes just solving an exercise isn't enough.
+/// We encourage you stay and mess around with the exercises, until you feel
```

Comment:
```suggestion
/// We encourage you to stay and mess around with the exercises, until you feel
```

Reply:
Fixed in 340875012539db8cdae6845c66a9bb6ee1b64751